src/test/scala/com/rhdzmota/examples/ServerSpecs.scala

```diff
@@ -1,57 +1,57 @@
 package com.rhdzmota.examples
 
 import akka.http.scaladsl.testkit.ScalatestRouteTest
-import com.rhdzmota.examples.model.{Bonjour, Greeting, Hello, Hola}
+import com.rhdzmota.examples.model.{Greeting, Salutation}
 import org.scalatest.{Matchers, WordSpec}
 
 class ServerSpecs extends WordSpec with Matchers with ScalatestRouteTest {
 
   "The Service" should {
 
-    "return a hello-world json (english) for GET request to /v1/hello" in {
-      Get("/v1/hello") ~> Server.route ~> check {
+    "return a hello-world json (english) for GET request to /v1.0.0/hello" in {
+      Get("/v1.0.0/hello") ~> Server.route ~> check {
```

Comment:
Would it be a good idea to include a package object to isolate constants like the API version?

Reply:
Good idea, haven't used package objects before. I'll add one for this use case following [this guideline](https://alvinalexander.com/scala/scala-package-objects-how-to-name-location-cookbook-recipe).  Thanks for the recommendation.